packages/shared/tsconfig.json

```diff
@@ -4,13 +4,6 @@
         "rootDir": "lib",
         "outDir": "dist"
     },
-    "references": [
-        {
-            "path": "../node-client"
-        },
-        {
-            "path": "../utils"
-        }
-    ],
+    "references": [{ "path": "../node-client" }, { "path": "../utils" }, { "path": "../records" }],
```

Comment:
we are fighting each other on .json files but prettier should have kicked in here

Reply:
what's the correct format?

Reply:
the one before your change, lint-staged should have handled this 🤔